src/components/SearchBar.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { Children, useState } from 'react';
+import { BrowserRouter as Router, Link, Route } from 'react-router-dom';
+import DetailedPage from '../pages/DetailedPage';
+
+export default function SearchBar() {
+  const [results, setResults] = useState([]);
+
+  const extractData = data => {
+    const extr = data.pop();
+    setResults(extr);
+  };
+
+  const updateInput = async event => {
+    await fetch(`http://localhost:3000/search?location=${event.target.value}`, { method: 'POST' })
+      .then(response => response.json())
+      .then(result => extractData(result))
+      .catch(error => console.log('error', error));
+  };
+
+  return (
+    <div className="search-bar container">
+      <input type="search" onChange={updateInput} placeholder={'find place..'} />
+      {!results ? 'results' : <ResultsList list={results} />}
+    </div>
+  );
+}
+
+function ResultsList(list) {
```

Comment:
1. please move component to separate file (one file - one component)
2. function component should contain `props` as param, not `list` or anything else